scripts/start_clean_app.sh

```diff
@@ -5,10 +5,11 @@ echo 'shutdown existing environment'
 docker-compose down
 
 echo 'remove docker data'
-rm -rf .docker/mongodb/data
+rm -rf .docker/mongodb/data/*
+mkdir -p .docker/mongodb/data
 
 echo 'run environment'
-docker-compose --env-file .env.dev up -d
+docker-compose --env-file ../.env.dev up -d
```

Comment:
This script is meant to be run from the root-directory of the project...

Reply:
But the script is put in a directory and it will run from there on out, so it can't directly use the root path of the project. At least it didn't work for me. Since it always run in its own directory path.